bin/p2-schedule/main.go

```diff
@@ -53,12 +53,14 @@ func main() {
 			log.Fatalf("Could not read manifest at %s: %s\n", manifestPath, err)
 		}
 		path := kp.IntentPath(*nodeName, manifest.ID())
-		if *hookTypeName != "" {
+		if *hookTypeName == "global" {
```

Comment:
If that were true then you would get a panic every time you used p2-schedule without a `hook-type`. I don't think kingpin supports this behavior so I propose a compromise:

- keep the existing `hook-type` flag and add a `global` setting so that it can be used to schedule global hooks
- add a hidden boolean option for just `hook` that always treats the manifest as a global hook, regardless of `hook-type`.

In our scripts we can start using the `--hook` option without showing it in `p2-schedule --help`. When we're satisfied with the global hook implementation, we kill the `hook-type` flag and unhide the `hook` flag. Thoughts?